package.json

```diff
@@ -6,8 +6,8 @@
   "dependencies": {
     "@material-ui/core": "^4.9.1",
     "@material-ui/icons": "^4.9.1",
-    "@twilio-labs/plugin-rtc": "^0.8.2",
-    "@twilio/conversations": "^1.1.0",
+    "@twilio-labs/plugin-rtc": "0.8.2",
+    "@twilio/conversations": "1.1.0",
     "@twilio/video-processors": "1.0.0",
```

Comment:
@timmydoza , that is correct. The increase in size is due to video processors dependencies. But we will shrink this down significantly in the upcoming release because we will be removing tfjs support. So I think we can leave this as is for now.

Reply:
Cool, sounds good!